homeassistant/components/tesla_wall_connector/strings.json

```diff
@@ -37,7 +37,7 @@
           "not_connected": "Vehicle not connected",
           "connected": "Vehicle connected",
           "ready": "Ready to charge",
-          "negociating": "Negociating connection",
+          "negotiating": "Negotiating connection",
```

Comment:
Please revert this change and fix it in a follow-up as it has nothing to do with this PR